### PR TITLE
QA Automation: patch the system before running the test

### PR DESCRIPTION
**Pull request by coolo**
This is to test Test Updates for SP1 and GA

### CODE REVIEW

**Comment by coolo** [comment]
@jtzhao what do you think?

**Comment by coolo** [comment]
reference: https://tortuga.suse.de/tests/123

@SeroSun ^

File: tests/qa_automation/patch_and_reboot.pm [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+use warnings;
+use testapi;
+
+sub run {
+    my $self = shift;
+    $self->system_login();
+
+    script_run("while pgrep packagekitd; do pkcon quit; sleep 1; done");
+
+    for my $var (qw(OS_TEST_REPO SDK_TEST_TEPO)) {
```
Comment (SeroSun) [inline tests/qa_automation/patch_and_reboot.pm]: Typo? s/SDK_TEST_TEPO/SDK_TEST_REPO/

**Comment by SeroSun** [comment]
Cool!
Will it patch all patches in url OS_TEST_REPO?

**Comment by jtzhao** [comment]
@coolo Looks good to me.

**Comment by mimi1vx** [comment]
:+1:

**Comment by coolo** [comment]
that variable only exists in qam-* tests - and I don't want to duplicate all regression tests.

**Comment by coolo** [comment]
and thanks for spotting the typo

**Pull request merged**